repos/system_upgrade/el7toel8/actors/peseventsscanner/actor.py

```diff
@@ -47,6 +59,9 @@ def process(self):
                 if action not in ('Present', 'Deprecated', 'Moved') and in_packages:
                     to_remove.update(in_packages)
                 if out_packages:
+                    out_repository = self.get_repository(event, 'out_packageset')
+                    for pkg in out_packages:
+                        pkg_repo[pkg] = out_repository
```

Comment:
there was a change in data format in PES (v2) repository is now part of the package, i.e.
`{"out_packageset": {
    "package": [
      {
        "name": "name",
        "repository": "repo"
      }
    ]
 }
`

Reply:
Thanks for pointing that out. Changed the code to handle V2 data, but, should we be backward compatible with V1 data? There is a expected scenario where the user will have updated actor and old (v1) data?

Reply:
we would need to have information from which version it is in the data first for backward compatibility to make sense but I don't think we will ever need to work with old data

Reply:
just a note: pes-data are still being downloaded from v1, I have a PR open to change it